engine/test_activation_shapes.py

```diff
@@ -0,0 +1,228 @@
+import matplotlib.pyplot as plt
```

Comment:
Lets move this file into the tests.functionality_tests folder

Reply:
Done